Cargo.toml

```diff
@@ -348,6 +348,17 @@ reqwest-retry = { git = "https://github.com/MaterializeInc/reqwest-middleware.gi
 # Note: All changes in our fork of tiberius should be pushed to the `mz_changes` branch.
 tiberius = { git = "https://github.com/MaterializeInc/tiberius", rev="64ca594cc22ed67d072c2d0110455da50539e1cd" }
 
+# Allows us to use bzip2-sys rather than the rust reimpl.
+# All changes should go to the `mz_changes` branch.
+# Once https://github.com/Nullus157/async-compression/pull/364 lands we can remove this.
+async-compression = { git = "https://github.com/MaterializeInc/async-compression.git", rev = "fe7411eb6104a02a89e2c3a76ab326dd6594214d" }
```

Comment:
The maintainer has replied with a way to do this without forking. We should enable the bzip feature we need from our side

Reply:
Without our fork, the bzip2 feature pulls in a crate with an unapproved license. Legal is reveiwing the license and I will follow up to remove our dep on the fork.